app/models/repo/task.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Repo
+  class Task < ApplicationRecord
+    extend FriendlyId
+    friendly_id :title, use: :slugged
+
+    belongs_to :challenge
+    belongs_to :dependent_task, class_name: 'Repo::Task', optional: true
+
+    validates :title, presence: true
+    validates :title, uniqueness: { case_sensitive: true }
+    validates :slug, presence: true, if: :persisted?
+    validates :slug, uniqueness: true, if: :persisted?
+
+    validates :start_at, comparison: { less_than: :result_at }, allow_nil: true, if: :result_at
+    validates :submit_at, comparison: { less_than: :start_at }, allow_nil: true, if: :start_at
```

Comment:
submit_at should be greater than the start. Timeline is:
start doing -> submit work -> show results

Reply:
added it here [update task validations](https://github.com/DevChallengeIT/devchallenge-platform/pull/19/commits/81e85721125592167133b5b939a6138f74cd7af0)